src/content/teams/12_formatting.mdx

```diff
@@ -0,0 +1,30 @@
+---
+name: Nix Formatting Team
+description: Maintains the official Nix formatter and standard format.
+members:
+- name: Silvan Mosberger
+  github: infinisil
+  title: Administrative leader, sponsored by Tweag and Antithesis
+- name: piegames
+  github: piegamesde
+  title: Main developer
+- name: Thomas Bereknyei
+  github: tomberek
+  title:
+- name: "0x4A6F"
+  github: "0x4A6F"
+  title:
+- name: Sergey Gulin
+  github: Sereja313
+  title: from Serokell
```

Comment:
```suggestion
  title: Sponsored by Serokell
```
Could you please also change this to "Sponsored by Serokell" to make it look more consistent with the others?